libpam/Makefile.am

```diff
@@ -40,3 +40,8 @@ libpam_la_SOURCES = pam_account.c pam_auth.c pam_data.c pam_delay.c \
 	pam_modutil_getgrgid.c pam_modutil_getpwuid.c pam_modutil_getgrnam.c \
 	pam_modutil_getspnam.c pam_modutil_getlogin.c pam_modutil_ingroup.c \
 	pam_modutil_priv.c pam_modutil_sanitize.c pam_modutil_searchkey.c
+
+# Pkg-config script.
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = pam.pc
+DISTCLEANFILES = $(pkgconfig_DATA)
```

Comment:
Since files created by `AC_CONFIG_FILES` are removed by `make distclean` automatically, no `DISTCLEANFILES` is needed.

Reply:
Thanks for the information, I probably should have read the documentation of AC_CONFIG_FILES too. (I understood that it generated the files given to it from the templates and didn't look further into it.)